sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventController.java

```diff
@@ -309,6 +390,40 @@ private EventDto findEvent(String uuid) {
 		return editView;
 	}
 
+	public CommitDiscardWrapperComponent<EventDataForm> getEventCreateComponent(EventDto event, boolean createSuperordinateEvent) {
```

Comment:
Either rename "event" to superOrSubordinateEvent OR use superordinateEvent and subordinateEvent parameters instead of the boolean (preferred).

Reply:
Also use EventReferenceDto instead